daemon/info_unix.go

```diff
@@ -14,6 +13,11 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	// RuncBinaryName is the name of the runc we are expecting
+	RuncBinaryName = "docker-runc"
```

Comment:
Let's clarify these constants for "docker-runc", I think this was also defined elsewhere for another purpose.

Reply:
The other place is a constant called "DefaultRuntimeName", which is the name of the default "runtime" use if no `--runtime` option is provided by the client. It's unfortunate that the name are equivalent, but I don't know what more to do. I think the comments are explicit enough, but I'm open to better wording

Reply:
Perhaps the other place could be `const DefaultRuntimeName = somepackage.RuncBinaryName` then? or does that cause include loops or other pain?

Reply:
Actually, this variable isn't even needed anymore, I'll remove it :innocent: